### ISSUE TITLE
restart option in start_slaves utility?

### ISSUE THREAD

**Issue by jknightusgs** [jtwhite79/pyemu#70]
Hi there,

I hacked the start_slaves utility to include an option to restart an aborted run. Seems to have worked and wonder if you think it's something worth including in your code.

I just added an option 'restart'=True (default value is False)...then under the master section, added an if/else to include ' \r' in the arg list if restart=True

` if restart:`
`    args = [exe_rel_path, pst_rel_path, "/r", "/h", ":{0}".format(port)]`
`else:`
`    args = [exe_rel_path, pst_rel_path, "/h", ":{0}".format(port)]`

**Comment by jtwhite79** [comment]
Hey Jake - sounds great!  Wanna submit a pull request?

**Comment by jknightusgs** [comment]
I wouldn't mind learning how...reading up on that now. It looks like I create a fork, make the changes there and then propose the changes be merged with the master branch? Seems straightforward enough...

also, I should probably include a few checks under the 'if restart:' block to confirm the requisite files are set up properly for a restarted run before the utility goes ahead with firing up all the workers (I had some issues with that after I got the /r command working). I think I had to run parrep to update the parval1 values in the pst file to those used in the last iteration before it was aborted.
